.github/workflows/vmtest.yml

```
@@ -21,6 +25,12 @@ jobs:
             echo "GIT_SHA=$GIT_SHA" >> $GITHUB_ENV
         - name: Install dependencies
           run: sudo apt -y install qemu-system-x86 curl
+        - name: Wait for build to succeed
+          uses: lewagon/wait-on-check-action@v1.2.0
+          with:
+            ref: ${{ github.ref }}
+            check-name: Build
+            wait-interval: 10
         - name: Download previously generated initramfs
           uses: dawidd6/action-download-artifact@v2
```

Could you pin the actions, please? `v2` is a branch

```suggestion
          uses: dawidd6/action-download-artifact@bd10f381a96414ce2b13a11bfa89902ba7cea07f # v2.2.3
```

---

Good shout! will do, will change it once I am done prototyping this :)